arch/arm/soc/nxp_imx/mcimx7_m4/soc.c

```diff
@@ -51,6 +51,30 @@ void SOC_RdcInit(void)
 	RDC_SetDomainID(RDC, rdcMdaM4, CONFIG_DOMAIN_ID, false);
 }
 
+#ifdef CONFIG_GPIO_IMX
+static void nxp_mcimx7_gpio_config(void)
+{
+
+#ifdef CONFIG_GPIO_IMX_PORT_1
+	RDC_SetPdapAccess(RDC, rdcPdapGpio1,
+		RDC_DOMAIN_PERM(CONFIG_DOMAIN_ID, RDC_DOMAIN_PERM_RW),
+		false, false);
```

Comment:
I apologize for not being more specific before. Please align lines 60-61 with the first argument in line 59. Do the same below for lines 69-70.

Reply:
You might want to check your vim settings and set tabstop=8. Sometimes the github diff viewer does funny things with alignment, but I checked out your code locally and it looks fine.